ff2mpv

```diff
@@ -4,7 +4,14 @@
 require "json"
 
 len = STDIN.read(4).unpack1("L")
-url = JSON.parse(STDIN.read(len))["url"]
+data = JSON.parse(STDIN.read(len))
+url = data["url"]
+opt = data["opt"]
+
+if opt == "video"
+  pid = spawn "mpv", "--no-terminal", "--", url, in: :close, out: "/dev/null", err: "/dev/null"
+elsif opt == "novideo"
+  pid = spawn "mpv", "--no-video", "--", url, in: :close, out: "/dev/null", err: "/dev/null"
```

Comment:
Shouldn't `--no-terminal` be passed in this case as well?

Reply:
(Same for the Python client).

Reply:
On my system there is no difference when called with or without the option, but I will add it for consistency.